### PR TITLE
[WIP] design updates

### PR DESCRIPTION
**Pull request by DLavin23**
@go-between/folks 

Working on some new thoughts for the chat messages.

- uses a "star" icon to pin messages
- moves the "commented during song" below the original message
- adds a nicer empty state while now videos are playing
- moves the activate user list to a horizontal list 

image

### CODE REVIEW

File: src/Room/Message.tsx [hunk 1]
```diff
@@ -25,9 +26,20 @@ const Message: React.FC<{ message: MessageType }> = ({ message }) => {
     user.id !== message.user.id || !message.song ? (
       ''
     ) : (
-      <Button sx={{ fontSize: 1 }} onClick={pinOrUnpin}>
-        {pinned ? 'unpin' : 'pin'}
-      </Button>
+      <Box
```
Comment (DLavin23) [inline src/Room/Message.tsx]: I'm not sure what I did wrong here, but the messages I pin in the chat don't seem to be showing up in the tab underneath the video.

Comment (trumanshuck) [reply src/Room/Message.tsx]: Even when you refresh the page?  It's not hooked up to a websocket.

Comment (DLavin23) [reply src/Room/Message.tsx]: Yes, even when I refresh the page. Was it previously hooked up to a websocket? I didn't think I changed anything other than the design/text of the component...

Comment (trumanshuck) [reply src/Room/Message.tsx]: It wasn't -- seems to be working as expected for me:
pinning

Did you do the ol' `bin/setup` after pinning got added (to run the necessary migrations in the api project)?

File: src/Room/Main.tsx [hunk 2]
```diff
@@ -36,13 +36,13 @@ const Main: React.FC<{ room: RoomType }> = ({ room }) => {
     >
       <Box>
         <QuickAdd />
-        <Heading
+        {/* <Heading
```
Comment (trumanshuck) [inline src/Room/Main.tsx]: Do we want this back?

File: src/Room/Main.tsx [hunk 3]
```diff
@@ -54,9 +54,14 @@ const Main: React.FC<{ room: RoomType }> = ({ room }) => {
           <Player />
         </Box>
 
-        <Box>
+        <Box
+          sx={{
+            // border: '1px solid',
```
Comment (trumanshuck) [inline src/Room/Main.tsx]: Nit:  Drop this comment.

File: src/Room/Message.tsx (hunk 1 above)
Comment (DLavin23) [reply src/Room/Message.tsx]: That's probably it.

**Pull request merged**